src/robusta/core/sinks/datadog/datadog_sink.py

```diff
@@ -77,6 +77,8 @@ def __enrichment_text(enrichment: Enrichment) -> str:
             elif isinstance(block, ListBlock):
                 lines.extend(block.items)
             elif isinstance(block, TableBlock):
+                if block.table_name:
+                    lines.append(f"%%%\n{block.table_name}\n%%%")
```

Comment:
out of curiosity what is %%%?

Reply:
It's datadog syntax for markdown in text block